images/base/files/usr/local/bin/entrypoint

```diff
@@ -298,9 +298,15 @@ fix_cgroup() {
   mount --make-rprivate /sys/fs/cgroup
   echo "${cgroup_subsystems}" |
   while IFS= read -r subsystem; do
-    mount_kubelet_cgroup_root "/kubelet" "${subsystem}"
-    mount_kubelet_cgroup_root "/kubelet.slice" "${subsystem}"
+    mount_kubelet_cgroup_root /kubelet "${subsystem}"
+    mount_kubelet_cgroup_root /kubelet.slice "${subsystem}"
   done
+  # workaround for hosts not running systemd
+  # we only do this for kubelet.slice because it's not relevant when not using
+  # the systemd cgroup driver
```

Comment:
functional change here

Reply:
@AkihiroSuda @kolyshkin can you please chime in here, I'm really in deep waters here and I think that a another pair of eyes will be nice for this change ... the cgroup glue keeps growing ...

Reply:
This doesn't work, wishful thinking :-)

https://github.com/kubernetes-sigs/kind/issues/2766#issuecomment-1126524575

I need to get a v1 host going again and take a peek at what exactly systemd is up to with the pseudo-subsystem.

Reply:
I setup colima on an old personal machine to debug, this _and_ a real slice works https://github.com/kubernetes-sigs/kind/pull/2767/commits/0ffcf8d6959f98ed038a925830b25f1f77480838.